src/LibChorus/LibChorus.csproj

```diff
@@ -36,6 +36,7 @@ See full changelog at https://github.com/sillsdev/chorus/blob/feature/nuget/CHAN
     <PackageReference Include="Icu4c.Win.Min" Version="59.1.7" />
     <PackageReference Include="L10NSharp" Version="4.0.2" />
     <PackageReference Include="NDesk.DBus" Version="0.15.0" />
+    <PackageReference Include="SIL.Chorus.Mercurial" Version="3.0.1-beta2" />
```

Comment:
There's a problem with version 3.0.1-beta2 of the NuGet package, which will probably fail to work properly on 32-bit Linux systems; see https://github.com/sillsdev/Mercurial4Chorus/pull/1 for details.

Reply:
done